updateHostsFile.py

```diff
@@ -24,7 +24,7 @@
 import time
 from glob import glob
 
-import lxml
+import lxml # noqa: F401
```

Comment:
@funilrys what is the significance of `# noqa: F401` here?

Reply:
@StevenBlack  `noqa` is a tag to tell flake8 to ignore error in that line (more info [here](http://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html#in-line-ignoring-errors))

If you remove that comments, flake8 produce https://travis-ci.org/funilrys/hosts/jobs/347500695#L59
```
./updateHostsFile.py:27:1: F401 'lxml' imported but unused
```

Explanation from https://github.com/StevenBlack/hosts/pull/520/commits/079d5ddd7f257c78ffd295743ff278626187be04 :
```
Please also note that I introduced that patch because
we do not directly use lxml but it is required by
BeautifulSup() to parse the HTML.
```